modules/integration/tests/src/test/java/org/wso2/appmanager/integration/test/cases/PublisherLogoutTestCase.java

```diff
@@ -59,7 +59,7 @@ public void startUp() throws Exception {
 
     @Test(description = TEST_DESCRIPTION)
     public void PublisherLogoutTestCase() throws Exception {
-        appmPublisherRestClient.webAppCreate(appName, context, appVersion, trackingCode);
+        appmPublisherRestClient.webAppCreate(appName, context, appVersion, trackingCode, userName);
```

Comment:
used webAppCreate method is generic, every time when you want to add new parameter you have to modify the arguments of the method as you have done. e.g if you want to create a app with role restricted using this method it can't be achieved .

I have added generic methods to create webapp , check the test case EditWebAppInCreatedStatusTestCase.java. You can create webapp in that way